www/include/views/graphTemplates/graphTemplates.php

```diff
@@ -1,54 +1,82 @@
+ *
+ * You should have received a copy of the GNU General Public License along with
  * this program; if not, see <http://www.gnu.org/licenses>.
- * 
- * Linking this program statically or dynamically with other modules is making a 
- * combined work based on this program. Thus, the terms and conditions of the GNU 
+ *
+ * Linking this program statically or dynamically with other modules is making a
+ * combined work based on this program. Thus, the terms and conditions of the GNU
  * General Public License cover the whole combination.
- * 
- * As a special exception, the copyright holders of this program give Centreon 
- * permission to link this program with independent modules to produce an executable, 
- * regardless of the license terms of these independent modules, and to copy and 
- * distribute the resulting executable under terms of Centreon choice, provided that 
- * Centreon also meet, for each linked independent module, the terms  and conditions 
- * of the license of that module. An independent module is a module which is not 
- * derived from this program. If you modify this program, you may extend this 
+ *
+ * As a special exception, the copyright holders of this program give Centreon
+ * permission to link this program with independent modules to produce an executable,
+ * regardless of the license terms of these independent modules, and to copy and
+ * distribute the resulting executable under terms of Centreon choice, provided that
+ * Centreon also meet, for each linked independent module, the terms  and conditions
+ * of the license of that module. An independent module is a module which is not
+ * derived from this program. If you modify this program, you may extend this
  * exception to your version of the program, but you are not obliged to do so. If you
  * do not wish to do so, delete this exception statement from your version.
- * 
+ *
  * For more information : contact@centreon.com
- * 
+ *
  */
 
 if (!isset($centreon)) {
     exit();
 }
 
-isset($_GET["graph_id"]) ? $cG = $_GET["graph_id"] : $cG = null;
-isset($_POST["graph_id"]) ? $cP = $_POST["graph_id"] : $cP = null;
-$cG ? $graph_id = $cG : $graph_id = $cP;
+$duplicationNumbers = [];
+$selectedGraphTemplates = [];
 
-isset($_GET["select"]) ? $cG = $_GET["select"] : $cG = null;
-isset($_POST["select"]) ? $cP = $_POST["select"] : $cP = null;
-$cG ? $select = $cG : $select = $cP;
+/*
+ * $_GET['graph_id'] $_POST['graph_id']
+ * id of the graph template
+ */
+$graph_id = filter_var(
+    $_GET['graph_id'] ?? $_POST['graph_id'] ?? false,
+    FILTER_VALIDATE_INT
+);
 
-isset($_GET["dupNbr"]) ? $cG = $_GET["dupNbr"] : $cG = null;
-isset($_POST["dupNbr"]) ? $cP = $_POST["dupNbr"] : $cP = null;
-$cG ? $dupNbr = $cG : $dupNbr = $cP;
+ /*
+  * Corresponding to the lines selected in the listing
+  * $_POST['select'] = [
+  *     'graphIdSelected' => 'duplicationFactor'
+  * ]
+  */
+if (isset($_POST['select']) && !empty($_POST['select'])) {
```

Comment:
if !empty also means it is isset
```suggestion
if (!empty($_POST['select'])) {
```